network-api/networkapi/settings.py

```diff
@@ -387,6 +388,7 @@
 
 # The SQS endpoint where we will send petition signups to
```

Comment:
minor nit: endpoints!

Reply:
not sure I understand the nit, it should be singular in this sentence, and then also a singular in the next one?